.github/workflows/02-scheduled-check.yaml

```diff
@@ -0,0 +1,20 @@
+name: Check Links 2
+run-name: ${{github.actor}} is checking links (exercise 2)
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+jobs:
+  check_links_2:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

Comment:
Any reason why you are using v3 for the actions rather than the more up to date v4? tech moves on, tutorials don't!